src/scripts/app.ts

```diff
@@ -106,35 +112,77 @@ async function addItems(chosedValue: number, imageArray: Basic[]) {
 		showToast('error', 'Cannot load images');
 	}
 
-	return createFindContainer(selectedImages);
+	createToFindItem();
 }
 
-async function createFindContainer(selectedImages: Array<object>) {
-	const findImageSection = imageArray.slice(0, 1);
-	const randomImg = Math.floor(Math.random() * selectedImages.length);
-	console.log(imgItemsArray);
-
-	findImageSection.forEach((imgItemsArray: Basic) => {
-		const findContainer: HTMLDivElement =
-			document.querySelector('.js-find-container')!;
-		const findTitle: HTMLElement = document.querySelector('.js-find-title')!;
-		const imgToFind = document.createElement('img');
-		imgToFind.id = imgItemsArray.id[randomImg];
-		imgToFind.src = imgItemsArray.urls.small;
-		imgToFind.classList.add('image-to-find');
-		findContainer.append(imgToFind, findTitle);
-	});
+function createToFindItem() {
+	const randomImg = Math.floor(Math.random() * imgItemsArray.length);
+	const originalImage = imgItemsArray[randomImg];
+
+	const findContainer: HTMLDivElement =
+		document.querySelector('.js-find-container')!;
+	const findTitle: HTMLElement = document.querySelector('.js-find-title')!;
+	const imgToFind = document.createElement('img');
+
+	imgToFind.id = originalImage.id;
+	imgToFind.src = originalImage.src;
+	imgToFind.classList.add('image-to-find');
+	findContainer.append(imgToFind, findTitle);
```

Comment:
I tutaj właśnie, imgItemsArray powinien zawierac tablice obrazkow, ktore maja swoje dane + doklejony ID. Jezeli go wylosujesz, to bierzesz to ID i nizej w linijce 129 np. dodajesz mu data-atrybut dzieki temu ID zostanie zapisane